### PR TITLE
Fix unnecessary scrollbar in legend

### PR DESCRIPTION
**Pull request by nxjiten**
@natabal 
I have fixed the unnecessary scrollbar in legends. Please have a look.

### CODE REVIEW

File: Graphs/AbstractGraph.js [hunk 1]
```diff
@@ -556,7 +556,8 @@ export default class AbstractGraph extends React.Component {
             legendWidth,
             legendHeight,
         } = this.getGraphDimension(label, data);
-
+        let legendContentHeight = 4;
```
Comment (natabal) [inline Graphs/AbstractGraph.js]: the only problem I have with this is that the height is static not based on the calculated legendHeight

**Comment by nxjiten** [comment]
@natabal 
I have calculated the height of the legend. Please have a look.

**Comment by nxjiten** [comment]
@natabal 
Ok I will re-calculate the height and width based on the legend height * number of lines.

**Comment by nxjiten** [comment]
@natabal 
I have calculated legend height dynamically using the container. Please have a look

**Pull request merged**